README.md

```diff
@@ -262,6 +262,35 @@ You can test it by running `freshclam` or `freshclam.exe` locally, where you've
 DatabaseMirror http://localhost:8000
 ```
 
+### Use docker
+
+Build docker image
+
+```bash
+docker build . --tag cvdupdate:latest
+```
+
+Run image, that will automaticly update databases in folder `/srv/cvdupdate`
+
+```bash
+docker run -d \
+  -v /srv/cvdupdate:/root/.cvdupdate/database \
+  cvdupdate:latest
+```
+
+Default update interval is `30 */4 * * *` (see [Cron Example](#cron-example))
+
+You may pass custum update interval in environment variable `CRON`
+
+For example - update every day in 00:00
+
+```bash
+docker run -d \
+  -v /srv/cvdupdate:/root/.cvdupdate/database \
+  -e CRON='0 0 * * *' \
+  cvdupdate:latest
+  ```
+
```

Comment:
I think it would be helpful to explain how to mount the logs directory as well, or provide some way to view the logs/output from the running process so you can tell it is working.

Reply:
Moved cvdupdate database and log folders to `/cvdupdate`, updated readme.

Reply:
Nice! I think that works!